testutil/src/data/keys.rs

```diff
@@ -12,10 +12,35 @@ pub const KEY1_RSA_KEYPAIR: &[u8] = include_bytes!("keys/key1.rsa.pk8");
 /// Test-only RSA public key generated from `key1.rsa.pk8`.
 #[rustfmt::skip]
 pub const KEY1_RSA_PUBLIC: &[u8] = include_bytes!("keys/key1.rsa.pub.pk8");
+/// RSA modulus of `key1.rsa.pk8`.
+#[rustfmt::skip]
+pub const KEY1_RSA_MOD: &[u8] = include_bytes!("keys/generated/key1.rsa.pub.mod");
+/// RSA exponent of `key1.rsa.pk8`.
+#[rustfmt::skip]
+pub const KEY1_RSA_EXP: &[u8] = include_bytes!("keys/generated/key1.rsa.pub.exp");
```

Comment:
This suggestion is outside the scope of this PR, but maybe have a single `#![rustfmt::skip]` at the top of the file?

Reply:
You're probably right, honestly. I may do that in a followup. I didn't supper care bc the file is machine-generated.